repository-structure.md

```diff
@@ -0,0 +1,72 @@
+
+Repositories MUST have these files with the specific content in the linked files, or a file with a
+link to the specified content with minimal exposition. These files MUST be at the root of the
+repository.
+
+-   [`LICENSE`](https://www.apache.org/licenses/LICENSE-2.0.txt) - https://www.apache.org/licenses/LICENSE-2.0.txt
+-   [`CODE_OF_CONDUCT.md`](https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct) - https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct
+-   [`SECURITY.md`](https://wiki.hyperledger.org/display/SEC/Defect+Response) - https://wiki.hyperledger.org/display/SEC/Defect+Response
+
+## Required Files with Variable Content
+
+Repositories MUST have these files. Named files MUST be at the root of the repository, and may have
+format suffixes such as `.md`, `.rst`, or `.txt`.
+
+-   `README` \
+    A description of the project and contain information or links to information such as
+    -   A reference to the Apache license (required).
+    -   The current and important past releases
+    -   Documentation for developers and users
+-   `MAINTAINERS` \
+    A list of all current maintainers with contact info. [A separate document](MAINTAINERS-guidelines.md)
+    covers the specifics.
+-   `CONTRIBUTING` \
+    Directions on how to contribute code to the project, or a pointer to the Wiki page with that information.
+-   `CHANGELOG` \
+    A human readable list of recent changes. Changes should at least include the current release. This
+    file may be maintainer curated or mechanically produced.
+-   Continuous Integration / Continuous Delivery (CICD) configurations \
+    Configurations needed to run CICD on Hyperledger provided systems.
+
+## Recommended
+
+Repositories SHOULD have these files. Named files SHOULD be at the root of the repository
+
+-   `NOTICE`
+    -   As per section 4 subsection d of the
+        [Apache License, Version 2](https://www.apache.org/licenses/LICENSE-2.0)
+-   Apache License Header information in each source code file. \
+    This SHOULD include the snippet `SPDX-License-Identifier: Apache-2.0` as part of the header.
+-   Build files consistent with the implementation language, such as...
+    -   For JavaScript/Node.js a `package.json` file
+    -   For Ruby a `Gemfile` file
+    -   For Java one of a Maven `pom.xml`, an Apache Ant `build.xml`, or a Gralde `build.gradle`
+        file
+    -   For Python `setup.py` and `requirements.txt` files
+    -   For Go `go.mod`, `go.sum` and as an option a Makefile
+    -   For Rust a `cargo.toml` file
+    -   For other languages, other standard build files a practitioner of the language would expect.
+-   Testing code \
+    Code to test the code in the repository (such as unit tests), in a location appropriate for the language.
+    \
+    Not all repositories can be tested (homebrew, docs), which is the only reason this is a SHOULD.
+
+## Prohibited
+
+Repositories MUST NOT have these files
```

Comment:
@hyperledger/tsc is there a defined process for releasing/publishing built artefacts? If not, maybe we can/should take a look at the current process and come up with recommendation.

Reply:
No.  We should form a task force, IMHO it's a problem.

Reply:
No action for this in this PR.